docs/spicedb-dedicated/dedicated-configuration.md

```diff
@@ -0,0 +1,54 @@
+# SpiceDB Dedicated Configuration
+
+The guide provides information about the configuration options available to you when deploying a SpiceDB Dedicated Permission System.
+
+## Configuration Options
+
+### Permission System Type
+
+<img src={require("/img/ps-type.png").default} alt="permission system type" />
+
+Please select the choice (Production or Development) that is appropriate for the Permission System you are deploying. This choice will not effect the performance or security of your Permission System, but may have impact your Permission System in the future, so please choose appropriately.
+
+### Datastore
+
+<img src={require("/img/datastore-config.png").default} alt="datastore configuration" />
+
+Your SpiceDB Dedicated environment can be provisioned with multiple isolated databases. If you have more than one, choose the database that is appropriate for the Permission System you are deploying.
+
+### Update Channel
+
+<img src={require("/img/channels-config.png").default} alt="channels config" />
+
+SpiceDB Dedicated has two update channels: `rapid` and `regular` . You can select the update channel either when you launch the Permission System or in the settings page after you’ve launched it.
+
+`rapid` - gets every release that is not a release candidate.
+
+`regular` - trails behind `rapid` by at least one release.
+
+After you’ve deployed your Permission System, you can choose to keep up to date with a channel automatically, or you can “pin” to a version by un-checking the automatic updates box. If you uncheck the automatic release box, you’ll still be able to update, but will have to pick the next version manually.
```

Comment:
Can we add a screenshot of the box?

Reply:
there already is a rapid vs regular box. i suspect you missed it?